gradle-plugin/build.gradle.kts

```diff
@@ -25,3 +26,20 @@ dependencies {
 tasks.withType<KotlinCompile> {
     kotlinOptions.jvmTarget = "1.8"
 }
+
+pluginBundle {
+    website = "https://github.com/mozilla-mobile/android-automation-tools/blob/master/gradle-plugin/README.md"
+    vcsUrl = "https://github.com/mozilla-mobile/android-automation-tools"
+    tags = listOf("mozilla", "apt", "android")
+}
+
+gradlePlugin {
+    plugins {
+        create("mozillaPlugin") { // this identifier is unused on remote
+            id = "org.mozilla.apt"
+            displayName = "Mozilla APT Plugin"
+            description = "A plugin for the Android Product Team (APT) at Mozilla"
+            implementationClass = "org.mozilla.apt.MozillaPlugin"
```

Comment:
Minor nit: I am not sure if the name should include "APT". This plugin looks like it could be useful to other teams at Mozilla building Android apps (e.g. AC, Lockbox, Taipei, Test Pilot, ..). And looking back I'm not even sure if the "APT" team will use that name forever. :)

Reply:
@pocmo Do you have an alternative suggestion? Maybe `org.mozilla.android.MozillaPlugin`?